src/backend/eventEmitters/EventEmitter.ts

```diff
@@ -119,9 +120,7 @@ export type EventDataMap = {
   [EventNames.LOG]: BudgetTrackingEvent
 };
 
-type EmptyEvents = EventNames.IMPORT_PROCESS_START | EventNames.IMPORT_PROCESS_END | EventNames.EXPORT_PROCESS_START | EventNames.EXPORT_PROCESS_END;
-
-export class BudgetTrackingEventEmitter extends Emittery.Typed<EventDataMap, EmptyEvents> {
+export class BudgetTrackingEventEmitter extends Emittery<EventDataMap> {
```

Comment:
The Emittery is working directly with the map now, and no need to divide between empty and non-empty events.